core/src/components/pagination/pagination.spec.ts

```diff
@@ -17,6 +18,7 @@ describe(`Pagination`, () => {
 		// Need to move those test to only ngBootstrapPagination tests
 		// Need to add test of the simple widget alone.
 		pagination.patch({pagesFactory: ngBootstrapPagination(0, false, true)});
+		expectedState = pagination.state$();
```

Comment:
If you want to manage the expectedState globally (as you do. Another solution would have been to crerate this variable in each test), it's better to do `expectedState = state` just after the following subscribe. (`pagination.state$()` do a subscribe/unsubscribe to get the value, this would avoid it)